libshaderc_spvc/src/spvc_private.h

```diff
@@ -26,16 +26,8 @@
 // GLSL version produced when none specified nor detected from source.
 #define DEFAULT_GLSL_VERSION 450
 
-struct shaderc_spvc_compiler {};
-
-// Described in spvc.h.
-struct shaderc_spvc_compilation_result {
-  std::string string_output;
-  std::vector<uint32_t> binary_output;
-  std::string messages;
-  shaderc_compilation_status status =
-      shaderc_compilation_status_null_result_object;
-  std::unique_ptr<spirv_cross::Compiler> compiler;
+struct shaderc_spvc_context {
```

Comment:
This is a struct with one member. Is it due to anticipating that we might want to add more to it in the future?

Reply:
I am going to be moving the messages string into it in a follow on patch